
public class ExceptionEx {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try{
			Exception e = new Exception("고의로 발생시킴.");	//예외 객체 만듬.
			throw e ;		//예외를 발생.
		//	throw new Exception("고의로 발생시켰음."); 위의 두줄을 한줄로 요약할 수 있다.
		}catch (Exception e) {
			System.out.println("에러 메시지 : " + e.getMessage());
			e.printStackTrace();
		}
		System.out.println("프로그램이 정상 종료되었습니다.");
	}

}
